Example/Controllers/SwiftUI/Features/Settings/Features/MiniAppSettingsContactsView.swift

```diff
@@ -169,12 +198,34 @@ extension MiniAppSettingsContactsView {
                 errorMessage += "Email id cannot be empty.\n"
             }
 
+            guard let allEmails = contactData.allEmailList else {
+                errorMessage += "Please correct and try again."
+                return errorMessage
+            }
+
+            for (idx, emailId) in allEmails.enumerated() {
+                if !emailId.isValueEmpty() {
+                    if !emailId.isValidEmail() {
+                        errorMessage += "Email \(idx + 1) is invalid.\n"
+                    }
+                } else {
+                    errorMessage += "Email \(idx + 1) cannot be empty.\n"
```

Comment:
```suggestion
                    errorMessage += "E-mail \(idx + 1) cannot be empty.\n"
```

Reply:
I think doing this will include primary email in the list of additional emails. Which will have the redundant data in the list. And may create the confusion when using the additional emails list. This will also change the behavior of the allEmailList as per my understanding. and this list will always have one value.

Reply:
Thank you :) I have update this in new commit.